src/domain/task-system/services/Cargo.toml

```diff
@@ -22,9 +22,11 @@ doctest = false
 
 
 [dependencies]
+database-common = { workspace = true }
 event-bus = { workspace = true }
 opendatafabric = { workspace = true }
 kamu-core = { workspace = true }
+kamu-datasets-services = { workspace = true }
```

Comment:
Why do we have to link to service implementation crate and not the domain crate?

Reply:
I only see imports like 

`use kamu_datasets_services::domain::{DatasetEnvVar, DatasetEnvVarService};`

Reply:
great catch. Thanks for pointing. Probably VScode autoimport did that and I missed to doublecheck. FIxed and double checked